.github/workflows/unit-tests.yml

```diff
@@ -305,6 +305,48 @@ jobs:
       group: ${{ github.event_name == 'push' && github.run_number || github.ref }}-musl
       cancel-in-progress: true
 
+  ASAN_UBSAN:
```

Comment:
I think you might be able to fit this into the existing `jobs: ubuntu:` matrix

```
        include:
          - name: "ASAN_UBSAN"
            env_file: ci/deps/actions-311-numpydev.yaml
            pattern: "not slow and not network and not single_cpu and not skip_asan"
            asan_options: "detect_leaks=0"
            meson_args: --config-settings=setup-args="-Db_sanitize=address,undefined"
            ...
```

Then fields will be accessible with `matrix.field` and will be empty by default